src/changed.js

```diff
@@ -36,7 +34,7 @@ async function changed({
   let _changed = [];
 
   for (let { dag } of packagesWithChanges) {
-    let name = await arePathsTheSame(dag.cwd, workspaceCwd) ? dag.packageName : path.basename(dag.cwd);
+    let name = dag.packageName;
```

Comment:
This loop doesn't seem necessary anymore:

```
let _changed = packagesWithChanges.map(({ dag }) => dag.packageName);
```

Reply:
ok let me try that